docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: "3"
```

Comment:
Really good to see that you managed to do the docker-compose file and it worked for you.